packages/example-generator/src/run.ts

```diff
@@ -71,6 +71,12 @@ async function generateExamples(tmpDir: tmp.DirResult): Promise<void> {
     });
 
     for (const mutator of MUTATORS) {
+      // We only want to apply the tutorial imports mutator to the tutorial template
```

Comment:
I think it's fine to just always apply this because everything generated from this uses `@osdk/examples.one.dot.one` so any submodule imports of `@osdk/examples.one.dot.one/ontology/objects` will not work regardless if it's this template or another one